packages/adapter-typegoose/README.md

```diff
@@ -0,0 +1,28 @@
+<p align="center">
+  <br/>
+  <a href="https://authjs.dev" target="_blank">
+    <img height="64px" src="https://authjs.dev/img/logo/logo-sm.png" />
+  </a>
+  <a href="https://typegoose.github.io" target="_blank">
+    <img height="64px" src="https://authjs.dev/img/adapters/typegoose.svg"/>
```

Comment:
This image should be added here https://github.com/nextauthjs/next-auth/tree/main/docs/static/img/adapters